store/potholes.js

```diff
@@ -1,11 +1,36 @@
 import axios from 'axios';
 
+//ACTION TYPES
+const GET_POTHOLES = 'GET_POTHOLES'
+
+
+//ACTION CREATORS
+
+const getPotholes = (potholes) => {
+  return {
+    type: GET_POTHOLES,
+    potholes
+  }
+}
+
+//THUNKS
+
+export const fetchPotholes = (lat, lon, latDelt, lonDelt) => {
+  return async dispatch => {
+    const potholes = await axios.get(`http://192.168.1.137:8080/api/potholes/${lat}/${lon}`);
```

Comment:
can we abstract the IP address into a process.env variable in secrets.js?

Reply:
We should confirm with chaz  if /lat/lon follows REST or if we should use query params

Reply:
changed to query params

Reply:
looking into making ip into a variable...necessary for this merge or can do on a later one?